graphite_api/render/glyph.py

```diff
@@ -12,7 +12,12 @@
 See the License for the specific language governing permissions and
 limitations under the License."""
 
-import cairocffi as cairo
+try:
+    import cairocffi as cairo
+except (NameError, ImportError, AttributeError):
+    CAIRO_DISABLED = True
```

Comment:
You could do `cairo = None` here instead of adding another symbol. Then checks for `CAIRO_DISABLED` would become `if cairo is None`.